Gemfile.jruby-2.6.lock.release

```diff
@@ -0,0 +1,929 @@
+PATH
+  remote: logstash-core-plugin-api
+  specs:
+    logstash-core-plugin-api (2.1.16-java)
+      logstash-core (= 8.6.1)
```

Comment:
@mashhurs remember to replace all 8.6.1 to 8.7.0

Reply:
Yup, here it is: https://github.com/elastic/logstash/pull/14876